contents/docs/integrate/feature-flags-code/_snippets/feature-flags-code-api.mdx

```diff
@@ -96,6 +96,170 @@ sendFlagsRequest();
 
 > **Note:** The `groups` key is only required for group-based feature flags. If you use it, replace `group_type` and `group_id` with the values for your group such as `company: "Twitter"`.
 
+#### Using evaluation tags and runtime filtering without SDKs
+
+When making direct API calls to the `/flags` endpoint, you can control which flags are evaluated using evaluation tags and runtime filtering.
+
+##### Evaluation tags
+
+To filter flags by evaluation environment, include the `evaluation_environments` field in your request body:
+
+<MultiLanguage>
+
+```shell
+curl -v -L --header "Content-Type: application/json" -d '  {
+    "api_key": "<ph_project_api_key>",
+    "distinct_id": "distinct_id_of_your_user",
+    "evaluation_environments": ["production", "web"]
+}' "<ph_client_api_host>/flags?v=2"
+```
+
+```python
+import requests
+import json
+
+url = "<ph_client_api_host>/flags?v=2"
+headers = {
+    "Content-Type": "application/json"
+}
+payload = {
+    "api_key": "<ph_project_api_key>",
+    "distinct_id": "user distinct id",
+    "evaluation_environments": ["production", "web"]
+}
+response = requests.post(url, headers=headers, data=json.dumps(payload))
+print(response.json())
+```
+
+```javascript
+const response = await fetch("<ph_client_api_host>/flags?v=2", {
+    method: "POST",
+    headers: {
+        "Content-Type": "application/json",
+    },
+    body: JSON.stringify({
+        api_key: "<ph_project_api_key>",
+        distinct_id: "user-distinct-id",
+        evaluation_environments: ["production", "web"]
+    }),
+});
+const data = await response.json();
+```
+
+</MultiLanguage>
+
+Only flags where at least one evaluation tag matches (or flags with no tags at all) will be returned. For example:
+- Flag with evaluation tags `["production", "api", "backend"]` + request with `["production", "web"]` = ✅ Flag evaluates ("production" matches)
+- Flag with evaluation tags `["staging", "api"]` + request with `["production", "web"]` = ❌ Flag doesn't evaluate (no tags match)
+- Flag with evaluation tags `["web", "mobile"]` + request with `["production", "web"]` = ✅ Flag evaluates ("web" matches)
+- Flag with no evaluation tags = ✅ Always evaluates (backward compatibility)
+
+##### Runtime detection
+
+Evaluation runtime (server vs. client) is automatically detected based on your request headers and user-agent. This determines which flags are available based on their runtime setting (server-only, client-only, or all).
+
+**How runtime is detected:**
```

Comment:
I think this snippet is a little too deeply nested. Maybe the flags code api should be it's own guide that we just link to. I'm concerned about how hard it'll be to following H2->H5 then get bolded text headers.

Reply:
Let me help with this separately in another PR. I think this is really out growing the snippet. I'll experiment with condensing this snippet down to a small example and linking out to a full guide.